templates/pages/setup/webhook/webhook.html.twig

```diff
@@ -104,6 +104,26 @@
                 e.originalEvent.formData.append('url', editors[0].getValue());
             }
         });
+        $('#{{ url_editor_container_id }}').closest('form').find('select[name="itemtype"]').on('change', (e) => {
+            // Get new Monaco suggestions from the server and then recreate the editor while preserving the value
+            $.ajax({
+                url: '{{ config('root_doc') }}/ajax/webhook.php',
+                data: {
+                    action: 'get_monaco_suggestions',
+                    itemtype: e.target.value
+                },
+                dataType: 'json',
```

Comment:
This option is required because the content type of the response is incorrect. Indeed, it is defined by the `header("Content-Type: text/html; charset=UTF-8");` code, but iy should be `Content-Type: application/json; charset=UTF-8` in the current case.
It is preferable to fix the response header to not have to use this option.

```suggestion
```